site/content/en/docs/Installation/helm.md

```diff
@@ -28,6 +28,19 @@ $ helm install --name my-release --namespace agones-system agones/agones
 _We recommend to install Agones in its own namespaces (like `agones-system` as shown above)
 you can use the helm `--namespace` parameter to specify a different namespace._
 
+When running in production, Agones should be scheduled on a dedicated pool of nodes, distinct from where Game Servers are scheduled for better isolation and resiliency. By default Agones prefers to be scheduled on nodes labeled with `stable.agones.dev/agones-system=true` and tolerates node taint `stable.agones.dev/agones-system=true:NoExecute`. If no dedicated nodes are available, Agones will
```

Comment:
Final thing - wrap this is a `feature` shortcode with a 0.8.0 publish date, so it's hidden until that date.

Details: https://agones.dev/site/docs/contribute/

Otherwise, LGTM!

Reply:
Done